.github/workflows/ci.yaml

```diff
@@ -39,7 +39,18 @@ jobs:
           # Bazelisk will download bazel to here
           XDG_CACHE_HOME: ~/.cache/bazel-repo
         run: |
+          set -o errexit
           npm pack
           cd e2e
           npm install $(find ../ -name 'aspect-build-webpack*.tgz')
           bazel --bazelrc=../.github/workflows/ci.bazelrc --bazelrc=../.bazelrc test //...
+          # make sure that webpack in worker mode builds in subsequent invocations.
+          echo console.log\(\"$(date)\"\) > ./test/module.js
+          if bazel test //...; then 
```

Comment:
So you are updating module.js, running the tests again and you're wanting a failure? And if there is a failure then the `then` clause won't execute? Am I reading this correct?

Reply:
Yes, correct. we don't want bazel to succeed which would mean that the webpack worker didn't catch the change.

Reply:
I'll explain it above the statement for future readers.